File: planner-todo/src/main/java/ru/javabegin/micro/planner/todo/PlannerTodoApplication.java
package ru.javabegin.micro.planner.todo;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.cloud.netflix.eureka.EnableEurekaClient;
import ru.javabegin.micro.planner.entity.Category;

@SpringBootApplication
@EnableEurekaClient
public class PlannerTodoApplication {

    public static void main(String[] args) {
        SpringApplication.run(PlannerTodoApplication.class, args);
    }

}

File: planner-todo/src/main/java/ru/javabegin/micro/planner/todo/PlannerTodoApplication.kt
package ru.javabegin.micro.planner.todo

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.cloud.context.config.annotation.RefreshScope
import org.springframework.cloud.netflix.eureka.EnableEurekaClient
import org.springframework.cloud.openfeign.EnableFeignClients
import org.springframework.context.annotation.ComponentScan
import org.springframework.data.jpa.repository.config.EnableJpaRepositories

@SpringBootApplication
@EnableEurekaClient
@ComponentScan(basePackages = ["ru.javabegin.micro.planner"])
@EnableJpaRepositories(basePackages = ["ru.javabegin.micro.planner.todo"])
@EnableFeignClients
@RefreshScope
class PlannerTodoApplication

    fun main(args: Array<String>) {
        runApplication<PlannerTodoApplication>(*args)
    }
